app/src/org/commcare/connect/network/ConnectNetworkService.kt

```diff
@@ -0,0 +1,14 @@
+package org.commcare.connect.network
+
+import okhttp3.ResponseBody
+import retrofit2.Call
+import retrofit2.http.POST
+import retrofit2.http.Query
+
+interface ConnectNetworkService {
+
+    @POST("users/heartbeat")
+    fun makeHeartbeatRequest(
```

Comment:
@OrangeAndGreen Wanted to prototype this along with this work, think we should be moving to this pattern for all connect requests.

Reply:
Agreed, this looks like a much better implementation.